pennylane/devices/qutrit_mixed/utils.py

```diff
@@ -33,7 +34,7 @@ def get_einsum_mapping(
         is_state_batched (bool): Boolean representing whether the state is batched or not
 
     Returns:
-        str: indices mapping that defines the einsum
+        str: Indices mapping that defines the einsum
```

Comment:
```suggestion
        str: indices mapping that defines the einsum
```
Keep consistent with rest of docs; no need for change